MangoPay/PayOut.php

```diff
@@ -7,6 +7,18 @@
  */
 class PayOut extends Transaction
 {
+	/**
+	 * The unique identifier of the bank account.
+	 * @var string
+	 */
+	public $BankAccountId;
+
+	/**
+	 * Custom description to appear on the user’s bank statement along with the platform name. The recommended length is 12 characters – strings longer than this may be truncated depending on the bank.
```

Comment:
```suggestion
	 * Custom description to appear on the user’s bank statement along with the platform name.
	 * The recommended length is 12 characters – strings longer than this may be truncated depending on the bank.
```